rust/integration-tests/sdk_expected_db_output_files/imported_testnet_txns/fungible_asset_processor/278556781/coin_supply.json

```diff
@@ -5,7 +5,6 @@
     "coin_type": "0x1::aptos_coin::AptosCoin",
     "supply": "18544645433533405815",
     "transaction_timestamp": "2022-10-13T01:46:57",
-    "transaction_epoch": 746,
-    "inserted_at": "2024-10-15T05:16:56.913064"
```

Comment:
weird. I thought these `inserted_at` fields were removed in a previous PR already

Reply:
this rebasing is tricky.. they are removed in the next stacked pr.